great_expectations/expectations/expectation.py

```diff
@@ -1181,6 +1180,30 @@ def validate_configuration(self, configuration: Optional[ExpectationConfiguratio
         return True
 
 
+class ColumnPairExpectation(TableExpectation, ABC):
```

Comment:
@edjoesu This class should probably be called `ColumnPairAggregateExpectation` (to sound harmoniously with the one we already have `ColumnPairMapExpectation`).  Thank you!

Reply:
@edjoesu We are probably also interested in having its own `is_abstract()` method for this class.